CHANGELOG.md

```diff
@@ -1,6 +1,28 @@
 Changelog
 ================================================================================
 
+2004
+--------------------------------------------------------------------------------
+
+### Main Changes
+
+* Updated to CoreMedia Content Cloud 10.2004.
+
+* Updated `ActionTestBaseConfiguration` to latest version of the 
+  `TranslatablePredicate` interface.
+
+* Removed `translatableExpressions` configuration from  
+  `TranslateGccConfiguration` which is now provided by the Blueprint's 
+  multi-site module.
+  
+* Fixed ISSUE-12 to allow the usage of other setting types within the facade. 
```

Comment:
For 1910 we referred to issues via GitHub's hash-syntax. Now, we have the prefix ISSUE here, which resembles to a JIRA issue. We should straighten reference to issues. Unfortunately, GitHub does not resolve any of the syntax to link to issues/PRs.

I would suggest sticking to hash-mark as this is well-known at GitHub. To emphasize, that this is an issue, we may use the FQN reference: `CoreMedia/coremedia-globallink-connect-integration#12`. As I assume, that GitHub won't resolve this either, just keep it as text then, or make it a Markdown link (which is unfortunately harder to read in plain-text mode).